### PR TITLE
i18n support and Chinese translation

### PR DESCRIPTION
**Pull request by Cirn09**
I added internationalization support for this script, and then added Chinese translation.

I also plan to realize the function of automatically assigning free cats and automatically constructing the best unicorn buildings, which may be completed in a few days later.

### CODE REVIEW

File: kitten-scientists.user.js [hunk 1]
```diff
@@ -133,6 +135,13 @@ var run = function() {
             'craft.limited': 'Crafting {0}: limited to be proportional to cost ratio',
             'craft.unlimited': 'Crafting {0}: unlimited',
 
+            'distribute.limited': 'Distribute to {0}: stop when reach max',
+            'distribute.unlimited': 'Distribute to {0}: unlimited',
+            'act.distribute': 'Dsitribute a kitten to {0}',
```
Comment (cameroncondry) [inline kitten-scientists.user.js:140]: Small typo in word "Distribute"

**Comment by cameroncondry** [comment]
@Cirn09 
Thank you for this addition!

I absolutely would love to get this feature pulled in as soon as possible. Give me some time to test this out, and with it being a while since I last played Kittens I would welcome anyone else with some free time to help test the features.

In the meantime, feel free to add yourself as a contributor, and we'll get this pulled in asap! Thanks again, really stellar addition!

**Comment by Cirn09** [comment]
I haven't updated the readme.md
Because writing a short essay in English will take a long time for me.

**Comment by cameroncondry** [comment]
No worries, it will get updated at some point. Just wanted to make sure your contribution was recognized.

I haven't seen any issues with this update, but will give it until tonight before I bring it all in.

**Comment by cameroncondry** [comment]
I'm going to pull this feature into the repository. Again, thank you for submitting this update, very excited that we now have some localization!

**Pull request merged**